docs/docs/tutorials/orca_k8s/orca_k8s_tutorial.md

```diff
@@ -0,0 +1,563 @@
+              "spark.kubernetes.file.upload.path":"/bigdl/nfsdata/"
+              }
+
+        init_orca_context(cluster_mode="k8s-cluster", num_nodes=2, cores=2, memory="2g",
+                          master="k8s://https://<k8s-apiserver-host>:<k8s-apiserver-port>", 
+                          container_image="intelanalytics/bigdl-k8s:latest",
+                          penv_archive="file:///bigdl2.0/data/sgwhat/environment.tar.gz",
+                          extra_python_lib="/bigdl/nfsdata/model.py", conf=conf)
+        ```
+
+        When running Orca programs on `k8s-cluster` mode, please use the following additional configuration propeties: 
+        * `spark.kubernetes.driver.volumes.persistentVolumeClaim.nfsvolumeclaim.options.claimName`: specify the claim name of `persistentVolumeClaim` with volumnName `nfsvolumeclaim` to mount `persistentVolume` into driver pod;
+        * `spark.kubernetes.driver.volumes.persistentVolumeClaim.nfsvolumeclaim.mount.path`: add volumeName `nfsvolumeclaim` of the volumeType `persistentVolumeClaim` to driver pod on the NFS path specified in value;
+        * `spark.kubernetes.executor.volumes.persistentVolumeClaim.nfsvolumeclaim.options.claimName`: specify the claim name of `persistentVolumeClaim` with volumnName `nfsvolumeclaim` to mount `persistentVolume` into executor pods;
+        * `spark.kubernetes.executor.volumes.persistentVolumeClaim.nfsvolumeclaim.mount.path`: add volumeName `nfsvolumeclaim` of the volumeType `persistentVolumeClaim` to executor pods on the NFS path specified in value;
+        * `spark.kubernetes.authenticate.driver.serviceAccountName`: the service account for driver pod;
+        * `spark.kubernetes.file.upload.path`: the path to store files at spark submit side in cluster mode;
+    
+    2. Using Conda to install BigDL and needed Python dependency libraries (see __Section 3__).
+
+    3. Pack the current activate Conda environment to an archive.
+        ```
+        conda pack -o environment.tar.gz
+        ```
+
+* On the __Host Node__:
+    1. Upload the Conda archive to NFS.
+        ```bash
+        docker cp <containerID>:/opt/spark/work-dir/environment.tar.gz /bigdl/nfsdata
+        ```
+    2. Upload the example Python file to NFS.
+        ```bash
+        docker cp <containerID>:/opt/spark/work-dir/train.py /bigdl/nfsdata
+        ```
+    3. Upload the extra Python dependency file to NFS.
+        ```bash
+        docker cp <containerID>:/opt/spark/work-dir/model.py /bigdl/nfsdata
+        ```
+
+Run the example on __Client Container__:
+```bash
+python /bigdl/nfsdata/train.py --cluster_mode k8s-cluster --remote_dir /bigdl/nfsdata
+```
+
+In the script:
+* `cluster_mode`: set the cluster_mode in `init_orca_context`.
+* `remote_dir`: directory on NFS for the dataset (see __Section 4__) and saving the model.
+
+__Note:__
+* It will return a <driver-pod-name> when the application is completed.
```

Comment:
![image](https://user-images.githubusercontent.com/21994046/189283321-caafc945-d1ba-466a-8efb-005e9611782a.png)
wrong layout

Reply:
Add some sample print logs of the application here